Documentation/GettingStartedWithTheMRTK.md

```diff
@@ -38,28 +38,121 @@ To get started with the Mixed Reality Toolkit you will need:
-    - (Optional) Microsoft.MixedRealityToolkit.Unity.Tools.unitypackage
-    - (Optional) Microsoft.MixedRealityToolkit.Unity.Examples.unitypackage
+    - (**_Optional_**) Microsoft.MixedRealityToolkit.Unity.Extensions.unitypackage
+    - (**_Optional_**) Microsoft.MixedRealityToolkit.Unity.Tools.unitypackage
+    - (**_Optional_**) Microsoft.MixedRealityToolkit.Unity.Examples.unitypackage
+    - (**_Optional_**, **_Experimental_**) Microsoft.MixedRealityToolkit.Unity.Providers.UnityAR.unitypackage
 
 The Mixed Reality Toolkit is now available for download on NuGet.org, for details see [MRTK NuGet Packages](MRTKNuGetPackage.md).
 
+#### Package contents
+
+**Foundation**
+
+The Microsoft.MixedRealityToolkit.Unity.Foundation package includes the core components required to create a mixed reality application.
+
+| Folder | Component | Description |
+| --- | --- | --- |
+| MixedRealityToolkit | | Interface and type definitions, base classes, standard shader. |
+| MixedRealityToolkit.Providers | | |
+| | [ObjectMeshObserver](SpatialAwareness/SpatialObjectMeshObserver.md) | Spatial awareness observer using a 3D model as the data. |
+| | OpenVR | Support for OpenVR devices. |
+| | WindowsMixedReality | Support for Windows Mixed Reality devices, including Microsoft HoloLens and immersive headsets. |
+| | WindowsVoiceInput | Support for speech and dicatation on Microsoft Windows platforms. |
+| MixedRealityToolkit.SDK | | |
+| | Experimental | Experimental features, including shaders, user interface controls and individual system managers. |
+| | Features | Functionality that builds upon the Foundation package. |
+| | Profiles | Default profiles for the Microsoft Mixed Reality Toolkit systems and services. |
+| | StandardAssets | Common assets; models, textures, materials, etc. |
+| MixedRealityToolkit.Services | | |
+| | [BoundarySystem](Boundary/BoundarySystemGettingStarted.md) | System implementing VR boundary support. |
+| | [CameraSystem](CameraSystem/CameraSystemOverview.md) | System implementing camera configuration and management. |
+| | [DiagnosticsSystem](Diagnostics/DiagnosticsSystemGettingStarted.md) | System implementing in application diagnostics, for example a visual profiler. |
+| | [InputAnimation](InputSimulation/InputAnimationRecording.md) | Support for recording head movement and hand tracking data. |
+| | [InputSimulation](InputSimulation/InputSimulationService.md) | Support for in-editor simulation of hand and eye input. |
+| | [InputSystem](Input/Overview.md) | System providing support for accessing and handling user input. |
+| | [SceneSystem](SceneSystem/SceneSystemGettingStarted.md) | System providing multi-scene application support. |
+| | [SpatialAwarenessSystem](SpatialAwareness/SpatialAwarenessGettingStarted.md) | System providing support for awareness of the user's environment. |
+| | [TeleportSystem](TeleportSystem/Overview.md) | System providing support for telporting (moving about the experience in jumps). |
```

Comment:
s/telporting/teleporting/gc

It'd probably also be worth doing a pass here using vscode+spellcheck addons to make sure we aren't missing anything else.